service/src/integrationTest/kotlin/fi/espoo/evaka/shared/job/ScheduledJobsTest.kt

```diff
@@ -1,8 +1,4 @@
-// SPDX-FileCopyrightText: 2017-2020 City of Espoo
-//
-// SPDX-License-Identifier: LGPL-2.1-or-later
-
```

Comment:
Mistakenly dropped?

Reply:
Oops :sweat_smile: 
The file has a lot of other unintended changes too...I'll redo this file